main.tf

```diff
@@ -219,6 +221,17 @@ resource "aws_security_group" "sgrp" {
     ]
   }
 
+  ingress {
+    from_port = "${var.SSH_from_port}"
+    to_port   = "${var.SSH_to_port}"
+    protocol  = "${var.SSH_protocol}"
+
+    cidr_blocks = [
+      "${var.dq_ops_ingress_cidr}",
+      "${var.peering_cidr_block}",
```

Comment:
Following a conversation with @david-wirelab, I don't think this is necessary, as we will only need to access SSH from the ops VPC (linux bastion)

Reply:
to specify - line 231: "${var.peering_cidr_block}",